test/feature/test_disk.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import torch
+
+from kornia.feature.disk import DISK, DISKFeatures
+
+
+class TestDisk:
+    def test_smoke(self, dtype, device):
+        disk = DISK().to(device, dtype)
+        inp = torch.ones(1, 3, 64, 64, device=device, dtype=dtype)
+        output = disk(inp)
+        assert isinstance(output, list)
+        assert len(output) == 1
+        for element in output:
+            assert isinstance(element, DISKFeatures)
```

Comment:
I think, this should work
```suggestion
        assert all(isinstance(element, DISKFeatures) for element in output)
```

Same for the others tests

Reply:
Done